workers/social/lib/social/models/account.coffee

```diff
@@ -367,6 +373,38 @@ module.exports = class JAccount extends jraphical.Module
           callback null, groups
 
 
+  fetchInviteGroups$: secure (client, options, callback) ->
```

Comment:
let's not expose `options` to client side, instead if you need to `fetchInvitedGroups` by `role` we can change that argument to `role`